src/style.css

```diff
@@ -0,0 +1,56 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.stars > [class^="stars--"] {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url("/images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--1 > .stars__star:nth-child(-n+1) {
+  background-image: url("/images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2 > .stars__star:nth-child(-n+2) {
+  background-image: url("/images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--3 > .stars__star:nth-child(-n+3) {
+  background-image: url("/images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 > .stars__star:nth-child(-n+4) {
+  background-image: url("/images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--5 > .stars__star:nth-child(-n+5) {
+  background-image: url("/images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
Check 2 point in [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)